app/controllers/data_services_controller.rb

```diff
@@ -2,8 +2,9 @@
 
 class DataServicesController < ApplicationController
   def index
+    params.delete(:filters) if params[:reset].present?
```

Comment:
This is very suspect... The reset type is HTML standard (https://www.w3schools.com/tags/att_input_type_reset.asp) why are we doing this?

Reply:
I think it's because the filter partial is within the wider index form, so by using the HTML reset standard it was wiping the search query from the results too.

Reply:
Ahh this may have been left over from one of our earlier attempts to get the reset working, I don't think we need it anymore 😄